ui-packages/packages/trusty/src/components/Organisms/Outcomes/Outcomes.tsx

```diff
@@ -32,10 +33,16 @@ type OutcomesProps =
     }
   | { outcomes: Outcome[]; listView?: false };
 
-const Outcomes = (props: OutcomesProps) => {
+const Outcomes: React.FC<OutcomesProps & OUIAProps> = (
+  props: OutcomesProps & OUIAProps
+) => {
+  const ouiaType = 'outcomes';
```

Comment:
I think you could have whole props as const here:
```suggestion
  const ouiaProps = componentOuiaProps(props.ouiaId, ouiaType, props.ouiaSafe);
```
and use as
```
      <section
        className="outcomes"
        {...ouiaProps}
      >
```